test/workflows/components/mnist.jsonnet

```diff
@@ -15,7 +15,7 @@ local defaultParams = {
   dataVolume: "kubeflow-test-volume",
 
   // Default step image:
-  stepImage: "gcr.io/kubeflow-ci/test-worker:v20190104-f2a1cdf-e3b0c4",
+  stepImage: "gcr.io/kubeflow-ci/test-worker/test-worker:v20190116-b7abb8d-e3b0c4",
```

Comment:
Curious: how is worker image being built? Is it automatically built or manually?

Reply:
Right now its all manual. We could build it on successful post submit like we do our examples we just need to add support for that.